src/float_cast.h

```diff
@@ -36,40 +36,24 @@
 */
 
 #include "config.h"
+#include <math.h>
 
 /*
 **	The presence of the required functions are detected during the configure
 **	process and the values HAVE_LRINT and HAVE_LRINTF are set accordingly in
 **	the config.h file.
 */
 
-#define		HAVE_LRINT_REPLACEMENT	0
```

Comment:
No, this is not going to be removed.

Reply:
Why? It is never used and not a standard macro. this header is also deprecated (I'd rather remove the whole header, see #68) so why keep it? What could it be used for?